packages/calcite-components/src/components/panel/panel.e2e.ts

```diff
@@ -456,51 +557,83 @@ describe("calcite-panel", () => {
     expect(await scrollEl.getProperty("scrollTop")).toBe(100);
   });
 
-  it("should close when Escape key is pressed and closable is true", async () => {
-    const page = await newE2EPage();
-    await page.setContent("<calcite-panel>test</calcite-panel>");
-    const panel = await page.find("calcite-panel");
-    const calcitePanelClose = await panel.spyOnEvent("calcitePanelClose");
-    const container = await page.find(`calcite-panel >>> .${CSS.container}`);
-    expect(await panel.getProperty("closed")).toBe(false);
-    expect(await container.isVisible()).toBe(true);
-    await container.press("Escape");
-    await page.waitForChanges();
-    expect(await panel.getProperty("closed")).toBe(false);
-    expect(await container.isVisible()).toBe(true);
-    panel.setProperty("closable", true);
-    await page.waitForChanges();
-    await container.press("Escape");
-    await page.waitForChanges();
-    expect(await panel.getProperty("closed")).toBe(true);
-    expect(await container.isVisible()).toBe(false);
-    expect(calcitePanelClose).toHaveReceivedEventTimes(1);
-  });
-
-  it("should not close when Escape key is prevented and closable is true", async () => {
```

Comment:
@driskull Due to the contents of `scrollingContentHtml`, this test is only covered under the scrolling use case. 

We could add coverage for this test for both scrolling and non-scrolling content by adding focusable elements to the content. Does that approach make sense? If so, would we want to keep scrolling content and non-scrolling content as the main test groups or do we need an additional group to cover scrolling with and without interactive content?